base_images/isce2/docker/Dockerfile

```diff
@@ -32,7 +32,7 @@ RUN sed -i -e 's/\/root/\/projects/g' /etc/passwd
 
 RUN conda install -y -c conda-forge pyOpenSSL=23.2.0 && \
     conda install -y -n base conda-libmamba-solver=23.7.0 && \
-    conda config --set solver libmamba
+    conda config --system --set solver libmamba
```

Comment:
We don't need this change since we aren't modifying isce2/ rgedi anymore. Do we want to just delete these files?

Reply:
Good question, at some point we need to take action on removing the `isce2` workspace.

Do we want to do that in this release with the short time we have or announce to the user group that it _WILL BE_ removed in the next release if no one speaks up. I vote for the latter.

@sujen1412 @marjo-luc @grallewellyn

Reply:
I don't know what the usage statistics are for these images. If no one is using them, then we should at least announce we aren't maintaining them beyond this release. I'm inclined to say we hold onto the latest versions of these images and freeze them, as @grallewellyn suggested, until we decide we no longer want to support them at all. At which point we can remove the Dockerfiles, images, and references in the dev files.